notebooks/301-tensorflow-training-openvino/301-tensorflow-training-openvino-nncf.ipynb

```diff
@@ -71,280 +71,223 @@
-    "        return {self._name: self._matches[-1]}\n",
-    "\n",
-    "    @property\n",
-    "    def avg_value(self):\n",
-    "        \"\"\"\n",
-    "        Returns accuracy metric value for all model outputs. Results per image are stored in\n",
-    "        self._matches, where True means a correct prediction and False a wrong prediction.\n",
-    "        Accuracy is computed as the number of correct predictions divided by the total\n",
-    "        number of predictions.\n",
-    "        \"\"\"\n",
-    "        num_correct = np.count_nonzero(self._matches)\n",
-    "        return {self._name: num_correct / len(self._matches)}\n",
-    "\n",
-    "    def update(self, output, target):\n",
-    "        \"\"\"Updates prediction matches.\n",
-    "\n",
-    "        :param output: model output\n",
-    "        :param target: annotations\n",
-    "        \"\"\"\n",
-    "        predict = np.argmax(output[0], axis=1)\n",
-    "        match = predict == target\n",
-    "        self._matches.append(match)\n",
-    "\n",
-    "    def reset(self):\n",
-    "        \"\"\"\n",
-    "        Resets the Accuracy metric. This is a required method that should initialize all\n",
-    "        attributes to their initial value.\n",
-    "        \"\"\"\n",
-    "        self._matches = []\n",
-    "\n",
-    "    def get_attributes(self):\n",
-    "        \"\"\"\n",
-    "        Returns a dictionary of metric attributes {metric_name: {attribute_name: value}}.\n",
-    "        Required attributes: 'direction': 'higher-better' or 'higher-worse'\n",
-    "                             'type': metric type\n",
-    "        \"\"\"\n",
-    "        return {self._name: {\"direction\": \"higher-better\", \"type\": \"accuracy\"}}"
-   ]
+    "quantized_model = nncf.quantize(\n",
+    "    ir_model,\n",
+    "    calibration_dataset,\n",
+    "    target_device=nncf.TargetDevice.CPU,\n",
+    "    subset_size=1000\n",
+    ")"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
   },
   {
-   "attachments": {},
    "cell_type": "markdown",
-   "metadata": {
-    "id": "mgaRN7mTXn30"
-   },
    "source": [
-    "## POT Optimization\n",
-    "\n",
-    "After creating the DataLoader and Metric classes, and defining the configuration settings for POT, we can start the quantization process."
-   ]
+    "Save quantized model to benchmark"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
   },
   {
    "cell_type": "code",
    "execution_count": null,
+   "outputs": [],
+   "source": [
+    "compressed_model_dir = Path(\"model/optimized\")\n",
+    "compressed_model_dir.mkdir(parents=True, exist_ok=True)\n",
+    "compressed_model_xml = compressed_model_dir / \"flower_ir.xml\"\n",
+    "serialize(quantized_model, str(compressed_model_xml))"
+   ],
    "metadata": {
-    "id": "RjywhxvqXn30"
-   },
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Compare File Size"
```

Comment:
I think you need to add some comments or change title to more relevant here. If I did not know context I don't understand what and why you do it.

Reply:
Make sense. It was changed.